runtime/src/main/scala/tailcall/runtime/model/ConstantType.scala

```diff
@@ -0,0 +1,16 @@
+package tailcall.runtime.model
+
+import tailcall.runtime.DirectiveCodec
+import zio.json.JsonCodec
+import zio.schema.annotation.caseName
+import zio.schema.codec.JsonCodec.jsonCodec
+import zio.schema.{DeriveSchema, Schema}
+
+@caseName("constant")
+final case class ConstantType(value: String)
```

Comment:
```suggestion
final case class ConstantType(value: Json)
```
Using Json will move the validation to the step where we read a config. You won't need to write code to validate if the string is Json.